examples/moviegen/README.md

```diff
@@ -0,0 +1,274 @@
+
+### Text-to-Image
+
+For more detailed instructions, please run `python inference.py --help`.
+
+```shell
+python inference.py \
+--config configs/inference/moviegen_t2i_256x256.yaml \
+--model.name llama-5B \
+--model.pretrained_model_path /path/to/llama-5B.ckpt \
+--text_emb.ul2_dir /path/to/ul2_embeddings \
+--text_emb.metaclip_dir /path/to/metaclip_embeddings \
+--text_emb.byt5_dir /path/to/byt5_embeddings \
+--image_size 256 455 \
+--batch_size 2
+```
+
+### Text-to-Video
+
+```shell
+python inference.py \
+--config configs/inference/moviegen_t2i_256x256.yaml \
+--model.name llama-5B \
+--model.pretrained_model_path /path/to/llama-5B.ckpt \
+--text_emb.ul2_dir /path/to/ul2_embeddings \
+--text_emb.metaclip_dir /path/to/metaclip_embeddings \
+--text_emb.byt5_dir /path/to/byt5_embeddings \
+--image_size 256 455 \
+--num_frames 32 \
+--batch_size 2 \
+--save_format mp4
+```
+
+## Training
+
+Movie Gen is trained jointly on images and videos in 4 stages:
+
+1. Training on images at 256 px resolution.
+2. Joint training on images and videos at 256 px resolution.
+3. Joint training at 768 px resolution.
+4. Fine-tune the model on high quality videos.
+
+Images are treated as single frame videos, enabling the use of the same model to generate both images and videos.
+Compared to video data, paired image-text datasets are easier to scale with diverse concepts and styles,
+and thus joint modeling of image and video leads to better generalization.
+
+To train Movie Gen, run the following commands:
+
+```shell
+scripts/stage1_train.sh # for stage 1 training
+scripts/stage2_train.sh # for stage 2 training
+```
+
+### Dataset Preparation
+
+Paths to videos and their corresponding captions should be stored in a CSV file with two columns: `video` and `caption`.
+For example:
+
+```text
+video,caption
+video_folder/part01/vid001.mp4,a cartoon character is walking through
+video_folder/part01/vid002.mp4,a red and white ball with an angry look on its face
+```
+
+### Generating Text Embeddings
+
+Due to the large memory footprint of the text encoders, the inference and training pipelines don't support generating
+text embeddings online. Please refer to the [Generating Text Embeddings](#generating-text-embeddings) section under the
+Inference section for details.
+
+### Cache Video Embedding (Optional)
+
+If you have sufficient storage budget, you can cache the video embeddings to speed up training by using the following
+command:
+
+```shell
+python inference_tae_enc.py \
+--tae.pretrained=/path/to/tae.ckpt \
+--tae.dtype=bf16 \
+--data.folder=/path/to/folder/with/videos/ \
+--output_path=/path/to/output/directory/ \
+--data.size=256 \
+--data.crop_size=[256,455]
+```
+
+### Performance
+
+| Model |      Context      | Jit level |   Stage   | Precision |          Resolution          | TAE Cache |       Batch size        | NPUs | Time (s/step) |                               Config                               |
```

Comment:
pls update the table format refer to wiki

Reply:
Updated